src/lib/serviceRoutinesV2/getAllSubscriptions.cpp

```diff
@@ -87,6 +87,7 @@ std::string getAllSubscriptions
   }
 
   std::string out;
+
```

Comment:
Please keep tidy files not affected by the PR.

```suggestion
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/397ae9e496a9ce542907b10016aa464b69ba71a3